Parsedown.php

```diff
@@ -75,6 +75,25 @@ function setUrlsLinked($urlsLinked)
 
     protected $urlsLinked = true;
 
+    function setSafeLinksEnabled($safeLinksEnabled)
+    {
+        $this->safeLinksEnabled = $safeLinksEnabled;
+
+        return $this;
+    }
+
+    protected $safeLinksEnabled = true;
+
+    protected $safeLinksWhitelist = array(
```

Comment:
In theory it would let an extension define additional protocols without having to worry about implementation of how to whitelist them